### PR TITLE
WINUI-3424-Addition of How-To page in avatar view UG

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Addition of How to page in the avatar view UG with badge state.

### CODE REVIEW

**Comment by MadhumithanJayapathi** [comment]
rebuild

File: winui/Avatar-view/how-to.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about using of the different states that exist in WinUi AvatarView control(SfAvatarView).
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# How to use different states
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/how-to.md:11]: @MadhumithanJayapathi How to use badge for WinUI AvatarView

File: winui/Avatar-view/how-to.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about using of the different states that exist in WinUi AvatarView control(SfAvatarView).
```
Comment (Manivannan-E) [inline winui/Avatar-view/how-to.md:4]: @MadhumithanJayapathi Learn about how to set the different states of badge for the WinUI AvatarView control.

File: winui/Avatar-view/how-to.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+control: AvatarView
+documentation: ug
+---
+
+# How to use different states
+
+## Set badge to avatar
+
+The [SfAvatarView](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.SfAvatarView.html) control provides support for [SfBadge](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Notifications.SfBadge.html) to notify users of new or unread messages, notifications, or the status of something.
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/how-to.md:15]: @MadhumithanJayapathi The SfAvatarView control supports various SfBadge states to notify users of new or unread messages, notifications, or status updates.

File: winui/Avatar-view/how-to.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+grid.Children.Add(ellipse);
+grid.Children.Add(textBlock);
+viewBox.Child = grid;
+badge.Content = viewBox;
+this.Content = badgeContainer;
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with Badge](avatarview_images/winui_badge_avatarview.png)
```
Comment (Manivannan-E) [inline winui/Avatar-view/how-to.md:81]: @MadhumithanJayapathi Change the image name.

File: winui/Avatar-view/how-to.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/how-to.md:11]: @Manivannan-E Changed the title as given.

File: winui/Avatar-view/how-to.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/how-to.md]: @Manivannan-E Changed the content as mentioned.

File: winui/Avatar-view/how-to.md (hunk 3 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/how-to.md:15]: @Manivannan-E Changed the content as mentioned.

File: winui/Avatar-view/how-to.md (hunk 4 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/how-to.md]: @MadhumithanJayapathi Changed the image name

File: winui/Avatar-view/how-to.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about how to set the different states of badge for the WinUI AvatarView control(SfAvatarView).
```
Comment (Manivannan-E) [inline winui/Avatar-view/how-to.md:4]: @MadhumithanJayapathi Change the title "Badge in WinUI AvatarView control | Syncfusion"

Comment (MadhumithanJayapathi) [reply winui/Avatar-view/how-to.md:4]: @Manivannan-E Changed the title as given.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**